docs/gettsim_objects/input_variables.md

```diff
@@ -231,9 +231,9 @@ Size of household dwelling in square meters
 
 Type: float
 
-(bewohnt_eigentum_hh)=
+(bewohnt_eigentum)=
 
-## `bewohnt_eigentum_hh`
+## `bewohnt_eigentum`
```

Comment:
I would have thought that this is always a hh-wide variable (i.e. if somebody in the hh owns the apartment, it would count as owner occupied for everybody - even if e.g. the children do not own it themselves)

- But you are thinking about a case in which one adult in the hh owns the apartment and another adult (most likely in another bg) pays rent to him/her?
- What do we expect the user to enter here for children?

Reply:
Tricky indeed. 

1. The typical case will be that there are 3 generations in one household, forming two `bg`en (1. grandparents, 2. parents + their kids).

   Say the grandparents own the house and there are no formal rent payments by the middle generation, let alone the kids. 

   Can we find out how that would work? E.g. 
   1. rent payments are imputed for the second `bg`
   2. interest payments + maintenance costs are distributed across all household members
   3. something else is being done

2. The other case would be when there are formal rent payments involved, but I think we can ignore that.

If 1.ii happens to be the case (what I would sort of hope), we can stick with the `hh` level, I think.

Reply:
Okay, then let's stick to `bewohnt_eigentum_hh` for now because the short-term goal is to implement the Kindergeldübertrag. I'll create an issue that addresses this.